src/vs/workbench/contrib/chat/browser/chatContentParts/toolInvocationParts/chatTerminalToolSubPart.ts

```diff
@@ -80,7 +80,8 @@ export class TerminalConfirmationWidgetSubPart extends BaseChatToolInvocationSub
 			verticalPadding: 5,
 			editorOptions: {
 				wordWrap: 'on',
-				readOnly: false
+				readOnly: false,
+				tabFocusMode: true
```

Comment:
Thanks! Later down in ~L224 we should update the editor options with `tabFocusMode: false` so that when the user expands the code block to multiline with "see more" they can use the tab key to indent

Reply:
That is false by default, so should work already

Reply:
Hmm not seeing line 224 in here, maybe I'm misunderstanding?

Reply:
found it